app/src/org/commcare/activities/components/ImageCaptureProcessing.java

```diff
@@ -132,6 +132,12 @@ public static void processImageChooserResponse(FormEntryActivity activity,
             showInvalidImageMessage(activity);
             return;
         }
+        if (!FileUtil.isSupportedMultiMediaFile(activity, selectedImage)) {
```

Comment:
is the same check implemented for other media capture widgets - audio/video ?

Reply:
Yeah it's present [here](https://github.com/dimagi/commcare-android/blob/master/app/src/org/commcare/activities/FormEntryActivity.java#L406)